platform/ext/target/nrf5340/target_cfg.h

```diff
@@ -0,0 +1,96 @@
+/*
+ * Copyright (c) 2017-2019 Arm Limited
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __TARGET_CFG_H__
+#define __TARGET_CFG_H__
+
+#include "platform/ext/common/uart_stdout.h"
+#include "tfm_plat_defs.h"
+
+
```

Comment:
You should add also definitions of `TFM_DRIVER_STDIO` and `NS_DRIVER_STDIO`, see [readme.rst](https://github.com/NordicPlayground/trusted-firmware-m/tree/master/platform/ext#target_cfgh).

Reply:
Yeap, that's correct; I only wanted to do these later, once we have the names of these stdio drivers. My intention for now is to push whatever is periperhal-agnostic :)

Reply:
@anangl is it OK to not define these macros now, since we don't know how they will be called and do it after we have defined the respective driver struct instances?

Reply:
It is OK. We could also start with both those macros just set to `Driver_USART0`.

Reply:
OK  just brought back these defines, @anangl